src/main/kotlin/vm/VM.kt

```diff
@@ -133,7 +134,7 @@ class VM(
                 }
                 O_JUMP -> {
                     val addr = next().address!!
-                    pc = addr
+                    if (addr >= 0) pc = addr else return VVoid
```

Comment:
This fixes a crash bug, when an 'if' statement is the last statement in a verb.  The final jump dest for skipping the 'else' is never reached by the compiler, so its dest is -1.  Any un-compiled jump dest can be assumed to be the end of the program.